docker-bake.hcl

```diff
@@ -0,0 +1,153 @@
+variable "repo" {
```

Comment:
What is this file for and should it be sorted somewhere other than root?

Reply:
It's simplest to keep this file in the root so we can simplify the docker build commands. We can move the Dockerfile into a nested folder though now.